types/prettier/parser-babel.d.ts

```diff
@@ -1,4 +1,16 @@
 import { Parser } from './';
 
-declare const parser: { parsers: { [parserName: string]: Parser } };
+declare const parser: {
+    parsers: {
+        babel: Parser;
+        'babel-flow': Parser;
+        'babel-ts': Parser;
+        json: Parser;
+        json5: Parser;
+        'json-stringify': Parser;
+        __js_expression: Parser;
+        __vue_expression: Parser;
+        __vue_event_binding: Parser;
```

Comment:
If we are listing private parsers, a new value `__babel_estree` was added #10910, but maybe to included in v2.3 yet.

Reply:
I have read exactly these parsers out with using `console.log(require('prettier/parser-babel'))`  
If they come out in a later version, I can add it later, okay?